src/sas/qtgui/Plotting/Plotter2D.py

```diff
@@ -222,6 +225,18 @@ def onClearSlicer(self):
         if self.slicer_widget:
             self.slicer_widget.setModel(None)
 
+    def onUnitsChange(self):
```

Comment:
This method is identical to that in `Plotter.py`. Can it be refactored to `PlotterBase`?